app/controllers/results_controller.rb

```diff
@@ -120,6 +121,11 @@ def edit
                              .order('group_name')
       @next_grouping = assigned_groupings.where('group_name > ?', @group.group_name).first
       @previous_grouping = assigned_groupings.where('group_name < ?', @group.group_name).last
+    elsif @result.is_a_review? && @current_user.is_reviewer_for?(@assignment, @result)
+      user_group = @current_user.grouping_for(@assignment.id)
+      assigned_prs = user_group.peer_reviews_to_others
+      @next_pr = assigned_prs.where('peer_reviews.id < ?', @result.peer_review_id).last
```

Comment:
Reuse the same variable names `@next_grouping` and `@previous_grouping`. This way you can eliminate the checks in the view.